frontend/src/routes/Infrastructure/ClusterTemplates/ClusterTemplatesPage/HelmRepositoriesTab.tsx

```diff
@@ -202,37 +195,44 @@ export const RowContext = React.createContext<{
 }>(undefined!);
 
 const HelmRepositoriesTab = () => {
-  const [repositories] = useHelmRepositories();
+  const [repositories, repositoriesLoaded, repositoriesError] = useHelmRepositories();
   const [repoIndex, repoIndexLoaded, repoIndexError] = useHelmRepositoryIndex();
   const { t } = useTranslation();
 
   return (
     <PageSection>
-      <RowContext.Provider
-        value={{
-          helmRepoIndexResult: [repoIndex, repoIndexLoaded, repoIndexError],
-          clusterTemplatesResult: useClusterTemplates(),
-        }}
+      <TableLoader
+        loaded={repositoriesLoaded}
+        error={repositoriesError}
+        errorId="helm-repositories-load-error"
+        errorMessage={t('The Helm repositories could not be loaded.')}
       >
-        <TableComposable
-          aria-label="Helm repositories table"
-          id="helm-repositories-table"
-          variant="compact"
+        <RowContext.Provider
```

Comment:
I think we dont need the context anymore, with the composable table we can easily just pass these as props to rows

Reply:
done